stylesheets/second.css

```diff
@@ -0,0 +1,102 @@
+
```

Comment:
For smaller projects that have the same overall theme, it's not necessary to do have multiple css files. I'm sure when you refactor you'll find that all of the selectors are the same and can be condensed into one file to be reused on multiple pages.